keyboards/treasure/type9s3/readme.md

```diff
@@ -0,0 +1,21 @@
+# Treasure TYPE-9 Series III
+
+![TYPE9SIII](https://i.imgur.com/lohJ0Dal.png)
+
+Treasure TYPE-9 Series III is a 3x3 Macropad.
+
+* Keyboard Maintainer: [Treasure](https://github.com/TreasureTypes)
+* Hardware Supported: Treasure TYPE-9 Series III PCB
+* Hardware Availability: [Treasure Store](http://treasuretypes.com)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make treasure/type9s3:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader :
+
+* **Physical reset button**: Briefly press the button on the back of the PCB
```

Comment:
as bootmagic is enabled, can a line regarding using bootmagic to enter bootloader be added?

Reply:
Ok i added a line, thanks